tests/test_losca_city_planning.py

```
@@ -0,0 +1,104 @@
+    assert (
+        parsed_items[0]["description"]
+        == "CITY PLANNING COMMISSION\r\nREGULAR MEETING AGENDA"
+    )
+
+
+def test_start():
+    assert parsed_items[0]["start"] == datetime(2024, 10, 24, 0, 0)
+
+
+def test_end():
+    assert parsed_items[0]["end"] == None
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)